firebase-abt/src/main/java/com/google/firebase/abt/FirebaseABTesting.java

```diff
@@ -140,6 +150,27 @@ public void removeAllExperiments() throws AbtException {
     removeExperiments(getAllExperimentsInAnalytics());
   }
 
+  /**
+   * Gets the origin service's list of experiments in the app.
+   *
+   * <p>Note: This is a blocking call and therefore should be called from a worker thread.
```

Comment:
is this a blocking call? should it be annotated with @WorkerThread?

Reply:
great call! updated the @workerthread annotation